packages/admin/dashboard/package.json

```diff
@@ -7,6 +7,9 @@
     "build": "tsup && node ./scripts/generate-types.js",
     "build:preview": "vite build",
     "preview": "vite preview",
+    "test": "vitest --run",
+    "i18n:validate": "node ./scripts/i18n/validate-translation.js",
+    "i18n:schema": "node ./scripts/i18n/generate-schema.js",
```

Comment:
I feel like it would be very easy to forget to run this. Should we add a test in our CI that after running this command there are no changes (i.e we haven't forgotten to run it after changing en.json)?

Reply:
That's what the test I added ensures. It checks if $schema.json === en.json. So if we make an update to en.json or the schema without updating the other the test will fail and block your PR.